### DISCUSSION TITLE
Drag to refresh feature

### DISCUSSION THREAD

**Discussion by pritkr** [ReVanced/revanced-discussions#264] [Question] [answered]
Drag to refresh is not working in the app. Please consider fixing it or creating a patch if it also doesn't work in youtube app

**Comment by SimonRalek** [comment]
I dont know what you downloaded, but the newest version refresh works fine.

**Comment by CnC-Robert** [comment]
This is because of the general ads patch, if you disable it pulling down to refresh will work again.

**Reply by oSumAtrIX** [reply]
An idea on why this could happen is because a sub view might be hidden but not the outer, entire view. Adjusting that in the patch could fix that issue.

**Reply by ronin08** [reply]
this bug is present from the start of vanced project! it would be nice if it gets patched.

**Reply by jeyjai** [reply]
It's been present since the start of revanced but not since the start of the original vanced so I'm guessing it's due to an old update to youtube that changed something

**Reply by CtrlasAltas** [reply]
On latest vanced sometimes it does work sometimes it doesn't, after restarting app, but on revanced this feature doesn't work at all. I also wanted to post this as a bug, but found out that's not only me having this issue. Please who has knowledge fix this bug. Thank you in advance... :P

**Reply by CtrlasAltas** [reply]
> This is because of the general ads patch, if you disable it pulling down to refresh will work again.

You are correct. Thanks. But then again I am seeing ads on revanced. So don't see a point disabling it.. :(

**Comment by UmRAtEJvgLtH** [comment]
I totally missed this thread and started a new one. Apologies for my lack of search skills.

The problem is not only with General Layout Ad removal option. When you enable Community Post removal, pull down to refresh on the subscriptions screen also no longer works.

**Comment by oSumAtrIX** [comment, marked as answer]
Moved to https://github.com/revanced/revanced-patches/issues/610
